rdt/transformers/categorical.py

```diff
@@ -264,7 +264,10 @@ def _fit(self, data):
         else:
             freq = data.value_counts(normalize=True, dropna=False)
 
+        nan_value = freq[np.nan] if np.nan in freq.index else None
         freq = freq.reindex(self.order).array
```

Comment:
When this reindexes, won't add None to the index as well since None should be in `self.order`?

Reply:
No because if there is `None` or `np.nan` in the data,  `data.value_counts` will return only `np.NaN` as index

Reply:
None is is self.order